### DISCUSSION TITLE
Additional Logged information

### DISCUSSION THREAD

**Discussion by joea99** [gramps-project/gramps-web-api#526] [General]
Requesting additional information be recorded in logs or perhaps an option for "verbose" logging?   I've not noticed that option if it is available.

I'd like to be able to note the user name at time of login and the email address of where notifications are sent, such as password reset, and new user registration.   Perhaps any administrative messages as well.

Is it appropriate to post these requests here, first, or just open an issue on github?

Gramps 5.1.6
Gramps Web API 1.6.0
Gramps Web Frontend 24.2.0
locale: en
multi-tree: false
task queue: true

**Comment by DavidMStraub** [comment]
Hi,

> Is it appropriate to post these requests here, first, or just open an issue on github?

Thanks for asking, yes, perfectly fine to use Github issues for feature requests. Since log messages are emitted by the backend, I'm shifting the issue there.

**Comment by DavidMStraub** [comment]
In general, there are several ways to increase log verbosity.

- If you use gunicorn for running the Web API, it has a `--log-level` command line switch
- If you use celery for running the task queue, it has a `--loglevel` command line switch
- If you use nginx as a reverse proxy, you can configure logging there as well

I suggest to play with these settings and see if it is sufficient.
